auto_tuning/proxy/src/proxy-runners/run-all.py

```diff
@@ -0,0 +1,78 @@
+import os, sys, inspect
+currentdir = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
```

Comment:
Just as a hint: Instead of inspect you can also use `__file__`. Inspect is fine as well, of course.

Reply:
done